src/zm_cluster_info.cpp

```diff
@@ -432,9 +433,14 @@ void zmClusterInfo::attributeDoubleClicked(const QModelIndex &index)
             const deCONZ::ZclAttribute &attr = cluster->attributes()[i];
             if (attr.id() == id)
             {
-                info->setAttribute(m_endpoint, m_clusterId, m_clusterSide, attr);
-                ok = true;
-                break;
+                QString mfc = "0x" + QString("%1").arg(attr.manufacturerCode(), 4, 16, QLatin1Char('0')).toUpper();
+
+                if (mfc == m_attrModel->data(m_attrModel->index(index.row(), 5)).toString())
```

Comment:
I'd prefer to not build string to compare :)

```
unsigned mfc = m_attrModel->data(m_attrModel->index(index.row(), 5)).toString().toUInt(nullptr, 0);
if (attr.manufacturerCode() == mfc)
{
}
```

It's a bit faster but more important doesn't rely on case sensitivity.

Reply:
Damn, this is what I wanted to have checked in addition. Will change it.

Reply:
Done